backends/p_sql.go

```diff
@@ -60,6 +70,17 @@ func (s *SQLProcessor) connect() (*sql.DB, error) {
 		Log().Error("cannot open database: ", err)
 		return nil, err
 	}
+
+	if s.config.MaxOpenConns != 0 {
+		db.SetMaxOpenConns(s.config.MaxOpenConns)
+	}
+	if s.config.MaxIdleConns != 0 {
+		db.SetMaxIdleConns(s.config.MaxIdleConns)
+	}
+	if s.config.MaxConnLifetimeSec != 0 {
```

Comment:
For MaxConnLifetimeSec, (after renamed to MaxConnLifetime) let's use `time.ParseDuration` to parse the string.
So then specifying time becomes more flexible
https://golang.org/pkg/time/#ParseDuration

Reply:
Good, thanks :)